transports/small-webrtc-transport/src/smallWebRTCTransport.ts

```diff
@@ -297,8 +329,43 @@ export class SmallWebRTCTransport extends Transport {
     logger.debug(`signalingState: ${pc.signalingState}`);
 
     pc.addEventListener("track", (evt: RTCTrackEvent) => {
-      logger.debug(`Received new track ${evt.track.kind}`);
-      this._callbacks.onTrackStarted?.(evt.track);
+      const streamType = evt.transceiver
+        ? evt.transceiver.mid === "0"
+          ? "microphone"
+          : evt.transceiver.mid === "1"
+            ? "camera"
+            : "screenVideo"
+        : null;
+      if (!streamType) {
+        logger.warn("Received track without transceiver mid", evt);
+        return;
+      }
+      logger.debug(
+        `Received new track ${evt.track.kind} ${evt.track.muted} ${evt.track.readyState}`,
+        evt,
+      );
+      this._incomingTracks.set(streamType, new WebRTCTrack(evt.track));
+      evt.track.addEventListener("unmute", () => {
```

Comment:
Have you tested in different browsers ? Do we always receive the `mute` or `unmute` event after we receive the event that a new track has started ?